docs/snippets/common/storybook-main-register-essentials-addon.js.mdx

```diff
@@ -0,0 +1,7 @@
+```js
+// .storybook/main.js
+
+module.exports = {
+  addons: ['@storybook/addon-essentials'],
+};
```

Comment:
@josh-degraw if you wouldn't mind updating this snippet and adding the stories field and the other addon that is installed as well. Turning this into:
```js
module.exports = {
  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
};

```
Reasoning behind this is to keep it consistent with the other examples used in our docs.

Reply:
Done